tests/integration/allocator_integration_tests.cpp

```diff
@@ -270,8 +271,8 @@ TEST(Allocation, DeallocateDifferent)
   auto& rm = umpire::ResourceManager::getInstance();
 
   auto alloc_one  = rm.getAllocator("HOST");
-  auto alloc_two = rm.makeAllocator<umpire::strategy::DynamicPool>(
-      "POOL_different_deallocate", alloc_one);
+  auto alloc_two = rm.makeAllocator<umpire::strategy::SizeLimiter>(
```

Comment:
This needs to be a SizeLimiter or else the assert on line 136 (another test) fails because it's checking the size of the record list.